packages/graphics/src/GraphicsData.ts

```diff
@@ -6,14 +16,21 @@
  */
 export class GraphicsData
 {
+    shape: IShape;
+    lineStyle: LineStyle;
+    fillStyle: FillStyle;
+    matrix: Matrix;
+    type: SHAPES;
+    points: number[];
+    holes: Array<GraphicsData>;
```

Comment:
These all seem like normal public variables so should have the `public` prefix

Reply:
Not, it is not pulicl api. It is internal.

Reply:
Whole class is public-y internal . Hacks are allowed, but its not a public API. I dont know whether to add modifiers here.